specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -490,6 +453,100 @@
       }
     }
   },
+  "x-ms-paths": {
+    "/custom/models?op=full": {
+      "get": {
+        "summary": "List Custom Models",
+        "description": "Get information about all custom models",
+        "operationId": "ListCustomModels",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "op",
```

Comment:
Should be required with a single enum value "full":
https://github.com/Azure/azure-rest-api-specs/blob/afe2cdb70a548ebf0348e433bf7d0437a2a8c986/specification/storage/data-plane/Microsoft.BlobStorage/preview/2019-12-12/blob.json#L161-L168